src/java/org/apache/cassandra/cql3/statements/schema/AlterKeyspaceStatement.java

```diff
@@ -62,6 +63,9 @@ public AlterKeyspaceStatement(String keyspaceName, KeyspaceAttributes attrs)
 
     public Keyspaces apply(Keyspaces schema)
     {
+        if (ClientWarn.instance.get() == null)
+            ClientWarn.instance.captureWarnings();
+        int previousWarningsSize = ClientWarn.instance.getWarnings() == null ? 0 : ClientWarn.instance.getWarnings().size();
```

Comment:
If we don't extract to numWarnings function, I think we need to change the naming a bit.
Size of warnings is better to be changed by num/number. Proposal:
`previousNumWarnings` and `newNumWarnings`